src/main/webapp/dev/js/main/adminSearch.es6

```diff
@@ -191,10 +191,11 @@ $(document).ready(() => {
                     `${crlf}${crlf}The following feedback session is ${sessionStatus}.${crlf}`
                     + `Course: [${courseId}][${courseName}]${crlf}Feedback Session Name: ${sessionName}${crlf}`
                     + `${crlf}The link to the feedback for the above session, please go to this Web`
-                    + ` address: ${relatedLink} %0D%0A%0D%0A*${uniqueLinkMessage}`;
+                    + ` address: ${relatedLink} ${crlf}${crlf}*${uniqueLinkMessage}`;
         }
 
-        const emailWrapper = `mailto:${studentEmail}?Subject=${subject}&body=${bodyHeader}${bodyContent}${bodyFooter}`;
+        const emailWrapper =
+                    encodeURI(`mailto:${studentEmail}?Subject=${subject}&body=${bodyHeader}${bodyContent}${bodyFooter}`);
```

Comment:
It's the individual components (subject and body) that need to be encoded, not the whole thing. This is to ensure that they do not contain characters that change the structure of the entire thing. For example, if `bodyHeader` is `Hello&world=`, then the entire `bodyContent` and `bodyFooter` will be missing from the email body. This is because they will not be part of the `body` param, but part of the `world` param instead.

Reply:
@whipermr5 Since I divided the body part further into three more component, so I thought it's better to `encode` just the mail URL instead of encoding all the body component separately and then add them to the URL.